curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f8f8bcd4370f6509078e.md

```diff
@@ -16,6 +16,8 @@ California<br />
 USA
 ```
 
+The `br` tags will allow each part of the address to be on their own line and are useful for presenting `address` elements properly.
```

Comment:
```suggestion
The `br` tags will allow each part of the address to be on its own line and are useful for presenting `address` elements properly.
```

Reply:
Just caught this after I gave my OK. "Each part" is singular so it should be "its".

Reply:
Good catch - dunno how I overlooked it.